generator/03-syndesis-db.yml.mustache

```diff
@@ -68,8 +117,17 @@
             value: ${POSTGRESQL_PASSWORD}
           - name: POSTGRESQL_DATABASE
             value: ${POSTGRESQL_DATABASE}
+          - name: POSTGRESQL_SAMPLEDB_PASSWORD
+            value: ${POSTGRESQL_SAMPLEDB_PASSWORD}
           image: ' '
           imagePullPolicy: IfNotPresent
+          lifecycle:
+            postStart:
```

Comment:
You said on chat, that `postStart` didn't work for you ? Does it work now ? (just curious)

Reply:
Yes it's working now. I had 2 issues, I tried running the sql at postStart but even the container was started, the database wasn't yet. So I added a bash shell to keep waiting till Postgres is up, and I needed that file to be executable. Turns out the octal number of 777 is stored as a decimal number in kubernetes config. This confused me for a bit. Also postStart has no logging unless you do redirect the output.